static/js/buddy_list.js

```diff
@@ -269,7 +269,11 @@ export class BuddyList extends BuddyListConf {
 
         // Add a fudge factor.
         height += 10;
-
+        if (this.keys.length === 0) {
+            this.container = $(this.container_sel);
+            this.container.append(`${_('No matching users')} `);
```

Comment:
It's better if you revamp it's styling according to `ListWidget` (If a certain query isn't found).

Reply:
can you please give a example.I am having a hard time finding it.

Reply:
The similar style used within `recent-topic` is a certain query is not found:

![Screenshot from 2021-04-14 11-53-56](https://user-images.githubusercontent.com/53977614/114663532-21fdf700-9d18-11eb-8dd0-c070626d7520.png)

Reply:
Thanks for helping.